sentry/README.md

```diff
@@ -77,3 +81,23 @@ For your security, the [`system.secret-key`](https://develop.sentry.dev/config/#
 ```
 helm upgrade ... --set system.secretKey=xx
 ```
+
+## Symbolicator
+
+For getting native stacktraces and minidumps symbolicated with debug symbols (e.g. iOS/Android), you need to enable Symbolicator via
+```yaml
+symbolicator:
+  enabled: true
+```
+
+However, you also need to share the data between sentry-worker and sentry-web via
```

Comment:
This is not the recommended way. 
Recommended way is to use something else than filestore, like S3 or GCS.

Please adapt your description with this ;)

Reply:
hm yes I agree. however S3 is not supported by the chart. Another option I forgot to mention is using an AWS EFS Storageclass, which supports ReadWriteMany.
I will update the doc and make the options more clear

Reply:
https://github.com/sentry-kubernetes/charts/blob/develop/sentry/values.yaml#L359
https://github.com/sentry-kubernetes/charts/blob/develop/sentry/values.yaml#L398-L405

S3 is supported by the chart 👍

Reply:
Hm ye I saw it in values but imho I didn't find a reference in the templates or did I miss it?
(Wouldn't have worked in my personal case tho as we cannot use IAM users/access-keys but need assumable roles and so on)

Reply:
@Mokto I updated again to be more precise about the storage options and caveats